.rubocop.yml

```diff
@@ -11,6 +11,7 @@ inherit_mode:
 AllCops:
   Exclude:
     - "db/migrate/*"
+    - "app/mailers/devise_mailer.rb"
```

Comment:
what was failing in this file? If there's a specific rule that doesn't make sense, I'd prefer we exclude that particular cop, either through this file, or with `# rubocop:disable` comment

Reply:
rubocop complained about this entire line:
https://github.com/heartcombo/devise/blob/45b831c4ea5a35914037bd27fe88b76d7b3683a4/lib/devise/mailers/helpers.rb#L17

because my devise mailer overrides the call made to method above, we need to get rubocop to ignore the problems it has with the line to make sure the override works properly - but I didn't want to make a rule for the whole app because it must complain for a reason (it doesn't like the fact that opts = {} is not in the end of the method call and it doesn't like &block - it prefers _block 

```
class DeviseMailer < Devise::Mailer
  def devise_mail(record, action, opts = {}, &block)
    initialize_from_record(record)
    view_mail("c978a30f-0578-4636-b07e-1e497c3b3893", headers_for(action, opts))
  end
end

```

Reply:
Ah yes, that makes sense. But as I mentioned, I'd prefer it if we were just disabling that single cop rather than AllCops for the file, similar to how to have disabled Rails cops at https://github.com/unboxed/bops/blob/master/.rubocop.yml#L15 - what is currently here disables all of rubocop for this file.